guides/micronaut-metrics/java/src/main/java/example/micronaut/DataPopulator.java

```diff
@@ -0,0 +1,31 @@
+package example.micronaut;
+
+import io.micronaut.context.annotation.Requires;
+import io.micronaut.context.event.StartupEvent;
+import io.micronaut.runtime.event.annotation.EventListener;
+import jakarta.inject.Singleton;
+
+import javax.transaction.Transactional;
+
+import static io.micronaut.context.env.Environment.TEST;
+
+@Singleton
+@Requires(notEnv = TEST)
+class DataPopulator {
+
+    private final BookRepository bookRepository;
+
+    DataPopulator(BookRepository bookRepository) {
+        this.bookRepository = bookRepository;
+    }
+
+    @EventListener
+    @Transactional
```

Comment:
you can make one comment that refers to 3 (or more) classes instead of spamming everyone. you can also commit more than one trivial change at a time instead of spamming everyone

Reply:
Sorry, I did separate commits in case you did not agree with any of them so that they could be easily reverted.